rfcs/glossary/Appendix C -- Glossary.md

```diff
@@ -172,6 +172,18 @@ accessing this HTTP endpoint _implicitly_ defines the GraphQL schema to use.
 > When using GraphQL over HTTP, it's common to encode the GraphQL request as
 > JSON.
 
+### error
+
+**Definition**: An error is an anomalous condition that breaks the execution of an algorithm.
+
+Errors are raised during the execution of an algorithm. Callers of the algorithm may handle errors and recover from the error.
+
+Note: errors can be thought as exceptions in languages that support them.
+
+### (GraphQL) field error
+
+**Definition**: A field error is an error raised during the execution of a particular field which results in partial response data. This may occur due to an internal error during value resolution or failure to coerce the resulting value.
```

Comment:
When the spec itself already includes an explicit hyperlinked definition for a term, is it necessary to duplicate it in the appendix manually?

I think many of the definitions in this appendix can be (and have been) migrated to the spec via `:: ` definitions. Perhaps working on a spec-md feature that lists all the definitions as part of the output would be a good idea.

Reply:
> I think many of the definitions in this appendix can be (and have been) migrated to the spec via :: definitions. 

100% agree. No need to manually maintain 2 source of truth. I forgot we had the `::` definitions.

I still see the Glossary as a good "incubating" place to discuss the terminology and allow PRs to be discussed/merged more easily while they make their way to something more official.